### DISCUSSION TITLE
exe链接超时，无法下载

### DISCUSSION THREAD

**Discussion by damifan-1** [chidiwilliams/buzz#990] [General]
exe链接超时，无法下载

**Comment by raivisdejus** [comment]
@damifan-1 Assuming you are asking for a download link to the installation exe you can try the following

1. Go to the latest development build section https://github.com/chidiwilliams/buzz/actions/workflows/ci.yml?query=branch%3Amain
2. Select the latest build
3. Scroll down to the Artifacts section
4. Download installation file. Please note, you need to be logged into the Github to see the download button
